### ISSUE TITLE
Google News panics upon loading

### ISSUE THREAD

**Issue by frewsxcv** [servo/servo#8093]
Steps to reproduce:

1. `./mach run -r http://news.google.com`
2. Wait ~10 seconds
3. Move mouse in Servo window
4. Servo crashes

[output log](https://gist.githubusercontent.com/frewsxcv/b3f9f214c85341538b59/raw/de9146e7050b421adc2698bc246043e09021597d/gistfile1.txt)

```
coreyf@frewbook-pro ~/D/r/servo (master) [141]> git log | head -1
commit 3f4ce134198e398fa94d1e754364b220037e6b49
```

```
host: x86_64-apple-darwin
```

**Comment by frewsxcv** [comment]
I just did a `git bisect` and traced it back to b649246fe2a14eacbf4e390c3161d980fc845773

**Comment by jdm** [comment]
That's an unfortunate result. What's the backtrace from lldb?

**Comment by jdm** [comment]
I swear I can read and click links. I am mollified that it's only a panic, not a segfault.

**Comment by frewsxcv** [comment]
Ah, sorry. I wasn't aware of the distinction between the terms 'crash' and 'panic'.

**Comment by frewsxcv** [comment]
Looks like this was potentially fixed with https://github.com/PistonDevelopers/image/issues/475. We just need an 'image' dependency upgrade....

**Comment by frewsxcv** [comment]
Fixed in #8224. Thanks @mbrubeck !
